tds/src/main/webapp/WEB-INF/classes/log4j2.xml

```diff
@@ -170,6 +170,11 @@
       <appender-ref ref="threddsServlet"/>
     </logger>
 
+    <!-- ASW SDK -->
+    <logger name="software.amazon.awssdk" level="warn" additivity="false">
+      <appender-ref ref="threddsServlet"/>
+    </logger>
```

Comment:
The security warning about authentication tokens should probably be left here, too in the comments because accidentally leaking that sort of stuff would be terrible.